components/fieldlabel/index.css

```diff
@@ -10,47 +10,62 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-fieldlabel.css";
-
 .spectrum-FieldLabel--sizeS {
-  @remapvars {
-    find: --spectrum-fieldlabel-s-;
-    replace: --spectrum-fieldlabel-;
-  }
+  --spectrum-fieldlabel-top-to-text: var(--spectrum-component-top-to-text-75);
+  --spectrum-fieldlabel-bottom-to-text: var(--spectrum-component-bottom-to-text-75);
+  --spectrum-fieldlabel-font-size: var(--spectrum-font-size-75);
+  --spectrum-fieldlabel-line-height: var(--spectrum-line-height-100);
+  --spectrum-fieldlabel-asterisk-gap: var(--spectrum-field-label-top-to-asterisk-small);
+  --spectrum-fieldlabel-side-padding-top: var(--spectrum-component-top-to-text-75);
+  --spectrum-fieldlabel-side-padding-right: var(--spectrum-spacing-100);
+  --spectrum-field-label-text-to-asterisk: var(--spectrum-field-label-text-to-asterisk-small);
 }
 
 .spectrum-FieldLabel--sizeM {
-  @remapvars {
-    find: --spectrum-fieldlabel-m-;
-    replace: --spectrum-fieldlabel-;
-  }
+  --spectrum-fieldlabel-top-to-text: var(--spectrum-component-top-to-text-75);
+  --spectrum-fieldlabel-bottom-to-text: var(--spectrum-component-bottom-to-text-75);
+  --spectrum-fieldlabel-font-size: var(--spectrum-font-size-75);
+  --spectrum-fieldlabel-line-height: var(--spectrum-line-height-200);
+  --spectrum-fieldlabel-asterisk-gap: var(--spectrum-field-label-top-to-asterisk-medium);
+  --spectrum-fieldlabel-side-padding-top: var(--spectrum-component-top-to-text-75);
+  --spectrum-fieldlabel-side-padding-right: var(--spectrum-spacing-200);
+  --spectrum-field-label-text-to-asterisk: var(--spectrum-field-label-text-to-asterisk-medium);
 }
-
 .spectrum-FieldLabel--sizeL {
-  @remapvars {
-    find: --spectrum-fieldlabel-l-;
-    replace: --spectrum-fieldlabel-;
-  }
+  --spectrum-fieldlabel-top-to-text: var(--spectrum-component-top-to-text-100);
+  --spectrum-fieldlabel-bottom-to-text: var(--spectrum-component-bottom-to-text-100);
+  --spectrum-fieldlabel-font-size: var(--spectrum-font-size-100);
+  --spectrum-fieldlabel-line-height: var(--spectrum-CJK-line-height-100);
+  --spectrum-fieldlabel-asterisk-gap: var(--spectrum-field-label-top-to-asterisk-large);
+  --spectrum-fieldlabel-side-padding-top: var(--spectrum-component-top-to-text-100);
+  --spectrum-fieldlabel-side-padding-right: var(--spectrum-spacing-200);
+  --spectrum-field-label-text-to-asterisk: var(--spectrum-field-label-text-to-asterisk-large);
 }
-
 .spectrum-FieldLabel--sizeXL {
-  @remapvars {
-    find: --spectrum-fieldlabel-xl-;
-    replace: --spectrum-fieldlabel-;
-  }
+  --spectrum-fieldlabel-top-to-text: var(--spectrum-component-top-to-text-200);
+  --spectrum-fieldlabel-bottom-to-text: var(--spectrum-component-bottom-to-text-200);
+  --spectrum-fieldlabel-font-size: var(--spectrum-font-size-200);
+  --spectrum-fieldlabel-line-height: var(--spectrum-CJK-line-height-200);
```

Comment:
same, lets double-check, maybe in a design workshop. I think we will discuss soon again the core-tokens for font-naming and weight and this might be a good additional question.